docs/source/release_notes/52_0.rst

```diff
@@ -0,0 +1,139 @@
+- Users sending SIGTERM to Avocado can now expect it to be properly handled.
+- `avocado.utils.linux_modules` functions now returns module name, size,
+  submodules if present, filename, version, number of modules using it,
+  list of modules it is dependent on and finally a list of params.
+- `avocado.TestFail`, `avocado.TestError` and `avocado.TestCancel` are
+  now public Avocado Test APIs, available from the main avocado namespace.
+- Support for new type of test status, `CANCEL`, and of course the
+  mechanisms to set a test with this status.
+- Introduction of the robot plugin, which allows Robot Framework tests
+  to be listed and executed natively within Avocado.
+- A brand new ResultsDB plugin.
+- When users run `avocado list --loaders ?` they used to receive a
+  single list containing loader plugins and test types, all mixed
+  together. Now users will get one loader listed per line, along with
+  the test types that each loader supports.
+- Variant-IDs created by the multiplexer are now much more meaningful.
+- The multiplexer now supports the use filters defined inside the YAML
+  files, and greatly expand its filtering capabilities.
+- Avocado now supports resuming jobs that were interrupted.
+- The HTML report now presents the test ID and variant ID in separate
+  columns, allowing users to also sort and filter results based on those
+  specific fields.
+- The HTML report will now show the test parameters used in a test when
+  the user hovers the cursor over the test name.
+- Avocado now reports the total job execution time on the UI, instead of
+  just the tests execution time.
+- The Avocado multiplex command has been renamed to variants.
+- Users will be given more information when a test reference is not
+  recognized by a given test loader.
+- Users can now choose to proceed with the execution of a job even if
+  one or more test references have not been resolved by one Avocado test
+  loader (AKA a test resolver).
+- The yaml-to-mux varianter implementation (the only one at this point)
+  is now an optional plugin.
+- Users can now choose in which order the job will execute tests (from
+  its suite) and variants.
+- Test methods on parent classes are now found upon the use of the new
+  `recursive` docstring directive.
+- The docstring directives now have a properly defined format.
+- Users can now see the tags set on `INSTRUMENTED` test when listing
+  tests with the `-V` (verbose) option.
+
+Those are the main features that are available for users. Besides that
+features, we have some statistics. When compared to the last LTS, we had:
+
+- 1187 commits (and counting).
+- 15 new versions.
+- 4811 more lines of Python code (+27,42%).
+- 1800 more lines of code comment (+24,67%).
+- 31 more Python files (+16,48%).
+- 69 closed GitHub issues.
+
+And that not all. We had great improvements in documentation, with new
+sections to Release Notes and Optional Plugins, very improved
+Contribution and Community Guide and much more.
```

Comment:
Also the `avocado-framework-tests` organization was founded to encourage companies to share Avocado tests.